testcontainers-spring-boot-parent/pom.xml

```diff
@@ -16,12 +16,12 @@
 
     <properties>
         <testcontainers.version>1.17.6</testcontainers.version>
-        <spring.boot.version>2.7.6</spring.boot.version>
-        <spring.cloud.version>2021.0.5</spring.cloud.version>
+        <spring.boot.version>3.0.2</spring.boot.version>
+        <spring.cloud.version>2022.0.0</spring.cloud.version>
```

Comment:
[Spring Cloud 2022.0.1](https://docs.spring.io/spring-cloud/docs/2022.0.1/reference/html/) is available.